modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -99,123 +95,340 @@ Status description:: error: connection exception - alias chains are not permitte
+
+=== 25N06
+
+Status description:: error: invalid transaction state - transaction start failed. Failed to start transaction. See debug log for details.
+
+=== 25N07
+
+Status description:: error: invalid transaction state - constituent transaction start failed. Failed to start constituent transaction. See debug log for details.
+
+=== 25N08
+
+Status description:: error: invalid transaction state - invalid transaction lease. The lease for the transaction is no longer valid.
+
+=== 25N09
+
+Status description:: error: invalid transaction state - internal transaction failure. The transaction failed due to an internal error.
+
+=== 25N11
+
+Status description:: error: invalid transaction state - conflicting transaction state. There was a conflict detected between the transaction state and applied updates. Please retry the transaction.
+
+=== 25N12
+
+Status description:: error: invalid transaction state - index was dropped. Index { $idx } was dropped in this transaction and cannot be used.
+
+=== 25N13
+
+Status description:: error: invalid transaction state - cannot access entity after removal. A { $entityType } was accessed after being deleted in this transaction. Verify the transaction statements.
+
+== Invalid transaction termination
+
+Invalid transaction termination errors occur when the transaction termination fails, such as when the transaction or constituent transaction fails to commit, or when the transaction termination fails to apply or append the transaction.
+
+=== 2DN01
+
+Status description:: error: invalid transaction termination - commit failed. Failed to commit transaction. See debug log for details.
+
+=== 2DN02
+
+Status description:: error: invalid transaction termination - constituent commit failed. Failed to commit constituent transaction. See debug log for details.
+
+=== 2DN03
+
+Status description:: error: invalid transaction termination - transaction termination failed. Failed to terminate transaction. See debug log for details.
+
+=== 2DN04
+
+Status description:: error: invalid transaction termination - constituent transaction termination failed. Failed to terminate constituent transaction. See debug log for details.
+
+=== 2DN05
+
+Status description:: error: invalid transaction termination - failed to apply transaction. There was an error on applying the transaction. See logs for more information.
+
+=== 2DN06
+
+Status description:: error: invalid transaction termination - failed to append transaction. There was an error on appending the transaction. See logs for more information.
+
+=== 2DN07
+
+Status description:: error: invalid transaction termination - inner transactions still open. Unable to commit transaction because it still have non-closed inner transactions.
+
+[[transaction-rollback]]
+== Transaction rollback
+
+Transaction rollback errors occur when the transaction is rolled back due to an error, such as when the transaction rollback fails, or when the constituent rollback fails.
```

Comment:
Yeah, I see. How about this one? This group of errors is a bit self-explanatory. 
```suggestion
Transaction rollback errors occur when there is a failure in a transaction or a constituent transaction rollback.
```

Reply:
Yeah that sounds clearer